bazel/docker/Dockerfile.clang-8.ubuntu18.04

```diff
@@ -0,0 +1,58 @@
+
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+# Docker file to setup build envrionment for nGraph-TensorFlow
+
+FROM ubuntu:18.04
+ENV http_proxy 'http://proxy-chain.intel.com:911'
```

Comment:
Remove these two lines. Update the instructions with information from here: https://github.com/tensorflow/ngraph-bridge/pull/212#issue-308729569